build.gradle

```diff
@@ -86,6 +86,12 @@ if (JavaVersion.current().isJava8Compatible()) {
     }
 }
 
+task currentJar(type: Copy){
```

Comment:
Think this should happen for `shadowJar` as well, since that is the main command for building Picard according to the README.